app/pb/auth.proto

```diff
@@ -73,8 +73,12 @@ service Auth {
     // Triggered when the user goes to the link sent in the forgot password email
   }
 
-  rpc CompleteChangeEmail(CompleteChangeEmailReq) returns (google.protobuf.Empty) {
-    // When a user opens a changed email confirmation, the frontend should call this function with the token
+  rpc ConfirmChangeEmailWithOldAddress(ConfirmChangeEmailWithOldAddressReq) returns (google.protobuf.Empty){
```

Comment:
`Confirm{New,Old}EmailAddress`?

Reply:
ConfirmOldEmailAddress sounds like you're confirming your old email address is correct for some reason. Ambiguous

Reply:
Maybe something like this?

```proto
message CompleteChangeEmailRes {
  // true if all emails confirmed and email changed, false otherwise
  bool success = 1;
  // one of these is true if success is false
  bool requires_confirmation_from_new_email = 2;
  bool requires_confirmation_from_old_email = 3;
}
```